tasks/38_max-product/max-product.js

```diff
@@ -0,0 +1,8 @@
+export function maxProduct(arr) {
+  let sum = [];
+  for (let i = 0; i < arr.length - 1; i++) {
+    sum[i] = arr[i] * arr[i + 1];
+  }
+  return Math.max(...sum);
```

Comment:
Fine solution but image what happen for very big number? Can you do it without Math.max and keeping values in array ?